### PR TITLE
GRO-277: Update /auctions header layout

### PR DESCRIPTION
**Pull request by The-Beez-Kneez**
This is for JIRA ticket [GRO-277]

Since the new copy text is rather long we wanted to  use this 50/50 header layout.

New Behavior:
Screen Shot 2021-04-07 at 2 40 11 PM

[GRO-277]: https://artsyproduct.atlassian.net/browse/GRO-277

### CODE REVIEW

**Review by jonallured — APPROVED** [review]
Looks so nice and clean!

File: src/v2/Apps/Auctions/AuctionsApp.tsx [hunk 1]
```diff
@@ -25,36 +31,44 @@ const AuctionsApp: React.FC<AuctionsAppProps> = props => {
...
-              height="15px"
-              width="14px"
-              top="3px"
-              left="3px"
-            />
-          </Text>
-        </RouterLink>
-      </HorizontalPadding>
+      <GridColumns>
+        <Column span={6}>
```
Comment (damassi) [inline src/v2/Apps/Auctions/AuctionsApp.tsx:35]: I'd double check mobile layout in chrome dev tools and make sure that we're collapsing this into a 12 column. So something llike `span={[12, 6]}` (and same below)

**Review by damassi — APPROVED** [review]
One tiny req, but otherwise LGTM 👍

File: src/v2/Apps/Auctions/AuctionsApp.tsx (hunk 1 above)
Comment (The-Beez-Kneez) [reply src/v2/Apps/Auctions/AuctionsApp.tsx]: Updated.

**Pull request merged**